src/LegacyRecord.php

```diff
@@ -399,4 +399,13 @@ public function bsonUnserialize(array $data): void
 
         $this->syncOriginalDocumentAttributes();
     }
+
+    /**
+     * Query model on database to retrieve an updated version of its attributes.
+     * @return self
```

Comment:
If it's defined return type, I believe that this annotation of the return isn't necessary.